extensions/csharp-v2/operation/method.ts

```diff
@@ -53,9 +53,9 @@ export class OperationMethod extends Method {
     // add parameters
     this.methodParameters = [];
 
-    const identity = new Parameter('identity', System.String);
+    const _identity = new Parameter('_identity', System.String);
```

Comment:
I'm not 100% sure how this "viaIdentity" branch used, but I suspect the string inside the Parameter() should still be 'identity' rather than '_identity'.